src/ralph/admin/fields.py

```diff
@@ -0,0 +1,109 @@
+# -*- coding: utf-8 -*-
+import re
+
+from django import forms
+from django.db import transaction
+from django.forms.util import ErrorList
+from django.utils.translation import ugettext as _
+
+
+class MultilineField(forms.CharField):
+    """
+    This field is a textarea which treats its content as many values seperated
+    by `separators`
+    Validation:
+        - separated values cannot duplicate each other,
+    """
+    widget = forms.Textarea
+    separators = ",|\n"
+
+    def __init__(self, allow_duplicates=True, *args, **kwargs):
+        self.allow_duplicates = allow_duplicates
+        super(MultilineField, self).__init__(*args, **kwargs)
```

Comment:
we should stick to one `super` convention - either `super(MultilineField, self).__init__(*args, **kwargs)` or super().__init__(*args, **kwargs)`